notebooks/community/sdk/sdk_pytorch_lightning_custom_container_training.ipynb

```diff
@@ -0,0 +1,1562 @@
+{
```

Comment:
<p>Tensorboard -&gt; TensorBoard</p><p>sentence case the title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/269/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
<p>Will do</p>

Reply:
<p>Close, missed these two instances:</p><p> </p><pre class="ql-syntax" spellcheck="false">"Your Tensorboard Resource Name or Tensorboard ID" 
</pre><p> </p>